src/main/cli/settings.c

```diff
@@ -1702,6 +1711,15 @@ const clivalue_t valueTable[] = {
     { "rangefinder_hardware", VAR_UINT8 | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_RANGEFINDER_HARDWARE }, PG_RANGEFINDER_CONFIG, offsetof(rangefinderConfig_t, rangefinder_hardware) },
 #endif
 
+// PG_OPTICALFLOW_CONFIG
+#ifdef USE_OPTICALFLOW
+    { "opticalflow_hardware",     VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OPTICALFLOW_HARDWARE }, PG_OPTICALFLOW_CONFIG, offsetof(opticalflowConfig_t, opticalflow_hardware) },
+    { "opticalflow_rotation",     VAR_INT16  | MASTER_VALUE ,              .config.minmaxUnsigned = {0, 359},               PG_OPTICALFLOW_CONFIG, offsetof(opticalflowConfig_t, rotation) },
+    { "opticalflow_lpf",          VAR_UINT16 | MASTER_VALUE ,              .config.minmaxUnsigned = {0, 10000},            PG_OPTICALFLOW_CONFIG, offsetof(opticalflowConfig_t, flow_lpf) },
+    { "opticalflow_flipX",        VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON },               PG_OPTICALFLOW_CONFIG, offsetof(opticalflowConfig_t, flipX) },
```

Comment:
I am not sure about the coordination system for the optical flow sensors in the market, so I made this the most flexible way.
But maybe I leave the rotation from 0 to 359 and one of the flips, like flip Y only.

Would this be a good idea?

Reply:
Other sensors should not be be a problem - either they use MSP and should use correct axis order(same as MT), or new driver will map axes correctly.
This is setting for user, there are multiple practical ways to mount it.
So, maybe, skip flip for now. If it is really necessary, it can be added easily.

Reply:
I kept the Y flip only, I myself need it for my setup

Reply:
Out of curiosity, can you explain your setup?

Reply:
The sensor is rotated 180 degrees, so I need to set the rotation to 270 and flip the Y-axis to align the coordinates of the sensor with the coordinates of the drone

Reply:
I agree with you; this will lead to standard coordinates on the sensor level, but is it worth making two rotations for each reading in the worst case?

Reply:
You can pre-calculate easily 2x2 transformation matrix. Applying it will take only 4 MUL/MAC instructions and you can factor all operations into it (rotation, flip, scaling).